hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceDetailRaw.tsx

```diff
@@ -59,44 +55,89 @@ export const InstanceDetailRaw: FunctionComponent<Props> = ({
     return (
         <>
             {showTitle && (
-                <Box mb={4}>
-                    <Typography variant="h5" color="secondary">
-                        <section>
-                            {`${formatMessage(MESSAGES.submissionTitle)} - ${
-                                data?.id
-                            }`}
-                            <IconButtonComponent
-                                url={`/forms/submission/instanceId/${data?.id}`}
-                                icon="remove-red-eye"
-                                tooltipMessage={MESSAGES.viewSubmissionDetails}
-                            />
-                        </section>
+                <Box display="flex" alignItems="center">
+                    <Typography variant={titleVariant} color="secondary">
+                        {`${formatMessage(MESSAGES.submissionTitle)} - ${
+                            data?.id
+                        }`}
                     </Typography>
+                    <Box
+                        display="inline-block"
+                        ml={1}
+                        sx={{
+                            display: 'inline-block',
+                            ml: 1,
+                            '& button a': {
+                                width: '30px',
+                                height: '30px',
+                            },
+                        }}
```

Comment:
sx is practical, but it has the same problem as `style`: it takes a lot of space in jsx. I would propose to put it in the guidelines to define `sx` styles in an object outside of the component